ARCs/arc-0079

```diff
@@ -0,0 +1,112 @@
+---
+arc: 79
+title: URI scheme, App NoOp call extension
+description: A specification for encoding NoOp Application call Transactions in a URI format.
+author: MG (@emg110)
+discussions-to: 
+status: Last Call
+last-call-deadline: 2024-10-30
+type: Standards Track
+category: Interface
+sub-category: General
+created: 2024-09-11
+extends: 26
+---
+
+## Abstract
+NoOp calls are Generic application calls to execute the Algorand smart contract ApprovalPrograms.
+
+This URI specification represents an extension to the base Algorand URI encoding standard ([ARC-26](./arc-0026.md)) that specifies encoding of application NoOp transactions into <a href="https://www.rfc-editor.org/rfc/rfc3986">RFC 3986</a> standard URIs.
+
+## Specification
+
+### General format
+
+As in [ARC-26](./arc-0026.md), URIs follow the general format for URIs as set forth in <a href="https://www.rfc-editor.org/rfc/rfc3986">RFC 3986</a>. The path component consists of an Algorand address, and the query component provides additional transaction parameters.
+
+Elements of the query component may contain characters outside the valid range. These are encoded differently depending on their expected character set. The text components (note, xnote) must first be encoded according to UTF-8, and then each octet of the corresponding UTF-8 sequence must be percent-encoded as described in RFC 3986. The binary components (args, refs, etc.) must be encoded with base64url as specified in <a href="https://www.rfc-editor.org/rfc/rfc4648.html#section-5">RFC 4648 section 5</a>.
+
+### ABNF Grammar
+
+```
+algorandurn     = "algorand://" algorandaddress [ "?" noopparams ]
```

Comment:
Maybe it's better to remove everything related to ARC-26, and something similar for the 78, so we only write what has been added. Let's discuss that during the next WC

```
algorandurn     = "algorand://" algorandaddress [ "?" algorandparams ]
algorandaddress = *base32
algorandparams  = algorandparam [ "&" algorandparams ]
algorandparam   = [ amountparam / labelparam / noteparam / assetparam / otherparam ]
amountparam     = "amount=" *digit
labelparam      = "label=" *qchar
assetparam      = "asset=" *digit
noteparam       = (xnote | note)
xnote           = "xnote=" *qchar
note            = "note=" *qchar
otherparam      = qchar *qchar [ "=" *qchar ]
```

Reply:
Good point, edited and updated!